app/controllers/users_controller.rb

```diff
@@ -61,11 +61,15 @@ def admin_redirect
   end
 
   def user_valid_destroy
-    @user.reviews.delete_all
+    @user.reviews.each do |review|
```

Comment:
Hound, you complicated. 

Hound is suggesting you do this: 

`@user.reviews.each(&:delete)`.